Dockerfile

```diff
@@ -18,7 +18,10 @@ FROM registry.ci.openshift.org/ocp/4.15:base
 ARG TAGS=""
 COPY --from=builder /go/src/github.com/openshift/machine-config-operator/instroot.tar /tmp/instroot.tar
 RUN cd / && tar xf /tmp/instroot.tar && rm -f /tmp/instroot.tar
+COPY --from=rhel9-builder /go/src/github.com/openshift/machine-config-operator/instroot/usr/bin/machine-config-controller /usr/bin/machine-config-controller.rhel9
```

Comment:
So confusingly, there's 2 files in the MCO: Dockerfile and Dockerfile.rhel7 . The name is not accurate, and it ends up that Dockerfile.rhel7 gets used for payload builds, and Dockerfile is used for ci builds (I think).

Should probably update both here

Reply:
I'll update both. Thanks!

Reply:
Done